diff --git a/node_modules/@react-three/drei/core/useEnvironment.js b/node_modules/@react-three/drei/core/useEnvironment.js
index 751601d..27f46c4 100644
--- a/node_modules/@react-three/drei/core/useEnvironment.js
+++ b/node_modules/@react-three/drei/core/useEnvironment.js
@@ -3,7 +3,7 @@ import { CubeReflectionMapping, EquirectangularReflectionMapping, sRGBEncoding,
 import { RGBELoader } from 'three-stdlib';
 import { presetsObj } from '../helpers/environment-assets.js';
 
-const CUBEMAP_ROOT = 'https://market-assets.fra1.cdn.digitaloceanspaces.com/market-assets/hdris/';
+const CUBEMAP_ROOT = 'https://raw.githack.com/pmndrs/drei-assets/456060a26bbeb8fdf79326f224b6d99b8bcce736/hdri/';
 function useEnvironment({
   files = ['/px.png', '/nx.png', '/py.png', '/ny.png', '/pz.png', '/nz.png'],
   path = '',
diff --git a/node_modules/@react-three/drei/helpers/environment-assets.js b/node_modules/@react-three/drei/helpers/environment-assets.js
index 789c6c5..69139c6 100644
--- a/node_modules/@react-three/drei/helpers/environment-assets.js
+++ b/node_modules/@react-three/drei/helpers/environment-assets.js
@@ -1,14 +1,14 @@
 const presetsObj = {
-  sunset: 'venice/venice_sunset_1k.hdr',
-  dawn: 'kiara/kiara_1_dawn_1k.hdr',
-  night: 'dikhololo/dikhololo_night_1k.hdr',
-  warehouse: 'empty-wharehouse/empty_warehouse_01_1k.hdr',
-  forest: 'forrest-slope/forest_slope_1k.hdr',
-  apartment: 'lebombo/lebombo_1k.hdr',
-  studio: 'studio-small-3/studio_small_03_1k.hdr',
-  city: 'potsdamer-platz/potsdamer_platz_1k.hdr',
-  park: 'rooitou/rooitou_park_1k.hdr',
-  lobby: 'st-fagans/st_fagans_interior_1k.hdr'
+  sunset: 'venice_sunset_1k.hdr',
+  dawn: 'kiara_1_dawn_1k.hdr',
+  night: 'dikhololo_night_1k.hdr',
+  warehouse: 'empty_warehouse_01_1k.hdr',
+  forest: 'forest_slope_1k.hdr',
+  apartment: 'lebombo_1k.hdr',
+  studio: 'studio_small_03_1k.hdr',
+  city: 'potsdamer_platz_1k.hdr',
+  park: 'rooitou_park_1k.hdr',
+  lobby: 'st_fagans_interior_1k.hdr'
 };
 
 export { presetsObj };
